CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### App Center
 
 * **[Fix]** Fix an `IncorrectContextUseViolation` warning when calculating screen size on Android 11.
+* **[Fix]** All SQL commands used in SDK are presented as raw strings, to avoid any possible static analyzer's SQL injection false alarms.
```

Comment:
```suggestion
* **[Fix]** All SQL commands used in SDK are presented as raw strings to avoid any possible static analyzer's SQL injection false alarms.
```